src/components/registerDelegate/steps/choose/choose.js

```diff
@@ -29,6 +29,7 @@ class Choose extends React.Component {
   }
 
   checkSufficientFunds(evt) {
+    Piwik.trackingEvent('Choose', 'button', 'onCheckSufficientFunds');
```

Comment:
The category would be more descriptive if it included the top component - based on the folder structure ('RegisterDelegate_Choose' instead of 'Choose'). And the event name to give a hint of the button text  ("Choose a name").
```suggestion
    Piwik.trackingEvent('RegisterDelegate_Choose', 'button', 'onChooseName');
```